### PR TITLE
lazyload app icons

### LINKED ISSUE #709: Use lazyload on apps/apps icons
Currently all loaded images are causing longer loading times and small performance issues on mobile devices. Loading the images via lazyload can fix that issue.

Maybe we should also think about a pagination in the future to make the DOM more performant (via infinite scroll, load more button)

### PR DESCRIPTION
**Pull request by zeke**
Possibly fixes #709 and #834 

Let's see how this feels. It's applied on these two pages:

https://electron-website-pr-879.herokuapp.com/ (scroll to the bottom)
https://electron-website-pr-879.herokuapp.com/apps

@groundwater

### CODE REVIEW

**Comment by zeke** [comment]
Stats on reduction in page size:

Route | Before | After
--- | --- | ---
/ | 1.7MB | 370KB
/apps | 2.2MB | 600KB

**Comment by vanessayuenn** [comment]
@zeke think you meant to reference this PR to fix #709? 😛

File: views/apps/index.html [hunk 1]
```diff
@@ -35,7 +35,8 @@ <h3>Categories</h3>
           <li class='listed-app'>
             <a href='/apps/{{this.slug}}' title='{{this.name}} - {{app.description}}'>
               <div class='listed-app-logo-wrapper'>
-                <img class='listed-app-logo' src='/node_modules/electron-apps/apps/{{this.slug}}/{{this.icon64}}' alt='{{app.name}}'>
+                <div class="listed-app-logo-placeholder placeholder" style="background: {{this.goodColorOnWhite}}"></div>
```
Comment (vanessayuenn) [inline views/apps/index.html]: I love how the placeholder makes use of the good color extracted from the logo! 👍 😍

Comment (zeke) [reply views/apps/index.html]: Thanks!

**Comment by zeke** [comment]
There's a small bug with the current implementation:

1. Visit https://electron-website-pr-879.herokuapp.com/apps?q=p2p
1. Erase the `p2p` from the search input
1. Behold the non-loading icons:

screen shot 2017-11-21 at 9 19 14 pm

**Comment by zeke** [comment]
With @groundwater's gzip improvement #883, the apps page is now down to 420KB. 💪

**Pull request merged**